packages/webcert/src/store/patient/patientReducer.ts

```diff
@@ -24,9 +24,6 @@ const errorReducer = createReducer(getInitialState(), (builder) => [
   builder.addCase(setStatus, (state, action) => {
     state.status = action.payload
   }),
-  builder.addCase(clearPatient, (state) => {
-    state.patient = undefined
```

Comment:
Isn't it better to keep this instead of removing it and adding the same in the middleware?

Reply:
I agree. I fixed this.